en/faq.md

```diff
@@ -0,0 +1,77 @@
+## FAQ
+
+### Why was the core of the SDK written in C++?
+
+The aim was to have an API in a language which is cross-platform and has many language bindings.
+Additionally, the library needs to be lightweight and fast, so it does not slow down for onboard usage at higher rate messaging.
+
+### Are multiple vehicles supported?
+
+Yes, the core C++ SDK is designed to support multiple vehicles. A vehicle is referred to a system in this SDK. The language wrappers so far only support one vehicle at a time. (However, nothing stops you from instantiating multiple copies of it to deal with multiple systems.)
+
+A system needs to have a specific MAVLink system ID but can consist of multiple components with different component IDs.
+An example would be a drone with a gimbal and a camera talking MAVLink with the same system ID but different component IDs.
+
+The limit is in theory 255 vehicles for system IDs ranging from 1 to 255.
+
+### Is MAVLink 1 supported?
+
+No. MAVSDK only supports [MAVLink 2.0](https://mavlink.io/en/guide/mavlink_2.html).
+
+### What sorts of vehicles are supported?
+
+The MAVSDK API is designed for interacting with *aircraft*. It has primarily been tested for use with multicopters, but also has basic support for fixed wing and [VTOL](../guide/vtol.md).
+
+The API *may* "work" with ground-based or other types of vehicles, but some methods will not make sense.
+This use-case is mostly unsupported and untested.
+
+### Does MAVSDK support indoor use?
+
+Indoor use is supported, however, some modes such as mission or position control are not available indoor, unless some additional positioning method is available (e.g. optical flow, visual-inertial odometry, a motion capture system, etc.).
+
+Note that PX4 currently does not support missions in "local coordinates" i.e. meters but only "global coordinates i. e. latitude/longitude.
+
+### What UAV flight stacks are supported?
+
+MAVSDK, so far, is optimised for use with the PX4 flight stack and all testing is done against PX4.
+
+While many features should work on other flight stacks there may be implementation differences at the [MAVLink microservices level](https://mavlink.io/en/protocol/overview.html) that mean not every API will work.
```

Comment:
My assumption right now is that it will not even connect to other stacks. If that is not true it would be good to say.

Reply:
It might or might not. I don't really know :shrug:.